languages/java/exercises/concept/inheritance/src/test/java/FighterTest.java

```diff
@@ -0,0 +1,55 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static junit.framework.TestCase.*;
+
+public class FighterTest {
+
+    @Ignore
+    @Test
+    public void testToString() {
+        Fighter warrior = new Warrior();
+        assertEquals("Fighter is a Warrior", warrior.toString());
+    }
+
+    @Ignore
```

Comment:
Usually we put a message with the `@Ignore` annotation like this
` @Ignore("Remove to run the test")`

Reply:
yes. Noted.